tox.ini

```diff
@@ -263,7 +267,8 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest sunpy/sunpy/io
+    pytest sunpy/sunpy/io \
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types:504"
```

Comment:
This will be fixed by the above request on the entry point.

Reply:
Made this change.